var/spack/repos/builtin/packages/gromacs/package.py

```diff
@@ -105,4 +117,20 @@ def cmake_args(self):
         else:
             options.append('-DGMX_USE_RDTSCP:BOOL=ON')
 
+        if '+subcounters' in self.spec:
+            options.append('-DGMX_CYCLE_SUBCOUNTERS:BOOL=ON')
+
+        if '+own_fftw' in self.spec:
+            options.append('-DGMX_BUILD_OWN_FFTW:BOOL=ON')
+
+        if '+mdrun_only' in self.spec:
+            options.append('-DGMX_BUILD_MDRUN_ONLY:BOOL=ON')
```

Comment:
@citibeth already mentioned this, but to make the build more deterministic in the event that future CMake defaults change, unconditionally append options like this, e.g.;

```python
options.append('-DGMX_BUILD_MDRUN_ONLY:BOOL=%s' % ('+mdrun_only' in spec))
```

Otherwise, if the build one day changes so that the default for this argument is `ON` instead of `OFF`, the variant may be set incorrectly.  Basically, try to be explicit.